### DISCUSSION TITLE
添加多语言支持

### DISCUSSION THREAD

**Discussion by KimYangOfCat** [FEDarling/fe-translate-weekly#241] [项目讨论]
为了保证站点内容来源的稳定性，希望添加多语言功能支持！将翻译前的英文内容同步保存在本站点，以防原始内容丢失。
目前已做了初步的尝试，请切换至[add_en](https://github.com/FEDarling/weekly-tracker/tree/add_en)分支，查看源码。
目前效果如下：

https://user-images.githubusercontent.com/32960305/161052512-5778c6ac-c0ed-49cb-ab4d-b71e3a9d8413.mp4

## 存在的问题
由于使用的主题不能完全支持多语言配置
![image](https://user-images.githubusercontent.com/32960305/161052953-271205ff-d124-4b61-a3dc-62051a506018.png)
所以目前功能是残缺的。后续如果想完全修复这个问题，可能需要修改主题源代码，工作量应该是不小的，故此先行在此讨论是否有必要支持多语言，然后再决定是否自主开发主题源代码？

欢迎任何讨论和意见！

**Comment by daodaolee** [comment]
个人感觉，目前还是没有做这块的必要....按目前的情况来看，翻译前的内容直接写入到待翻译的模板里，应该不会出现原始内容丢失的问题，而且我感觉如果我是读者，我并不会很在意原文到底是什么样的。🤔

**Reply by KimYangOfCat** [reply]
明白了，那暂时不做了吧
